libnmstate/schemas/operational-state.yaml

```diff
@@ -63,7 +63,6 @@ definitions:
         - down
     mac-address:
       type: string
-      pattern: "^([a-fA-F0-9]{2}:){5,31}[a-fA-F0-9]{2}$"
```

Comment:
@tyll pointed out correctly that it makes sense to keep the maximum length and the format validated.
If there is no explicit reason, I would use a minimum of 4 octets.

Reply:
Done, thank you!